### DISCUSSION TITLE
How long the Fairphone 2 will be supported?

### DISCUSSION THREAD

**Discussion by PetersOtto** [Divested-Mobile/DivestOS-Build#370] [General]
I'm really glad that I can continue to use my old Fairphone 2 with DisestOS.
The DivestOS custom rom for the Fairphone 2 is based on Android 11. Can anyone tell me how long the Fairphone 2 will be supported?

**Comment by SkewedZeppelin** [comment]
I can't put a number to it, but I recommend you do get a newer phone like the Pixel 6 or newer.

**Comment by PetersOtto** [comment]
Oh, that doesn't sound like much longer. &#128556;
I already have a newer phone, I only really use the FP2 to listen to music (Spotify) when I'm working in the garden or on the car (AndrOBD). But it's great for that in combination with DivestOS.

Thank you for maintaining DivestOS!
